src/pages/trade/HorizontalPanel.tsx

```diff
@@ -343,9 +347,41 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen, handleStartResize }:
     ]
   );
 
+  const paymentsTabItem = useMemo(
+    () => ({
+      asChild: true,
+      value: InfoSection.Payments,
+      label: stringGetter({ key: STRING_KEYS.PAYMENTS }),
+
+      content: (
+        <FundingPaymentsTable
+          columnKeys={
+            isTablet
+              ? [
+                  FundingPaymentsTableColumnKey.Time,
+                  FundingPaymentsTableColumnKey.Payment,
+                  FundingPaymentsTableColumnKey.Rate,
+                ]
+              : [
+                  !showCurrentMarket && FundingPaymentsTableColumnKey.Market,
+                  FundingPaymentsTableColumnKey.Time,
+                  FundingPaymentsTableColumnKey.Side,
+                  FundingPaymentsTableColumnKey.OraclePrice,
+                  FundingPaymentsTableColumnKey.Size,
+                  FundingPaymentsTableColumnKey.Payment,
+                  FundingPaymentsTableColumnKey.Rate,
+                ].filter(isTruthy)
+          }
+          initialPageSize={initialPageSize}
```

Comment:
we might need to add a prop to filter data to just the currentMarket or all markets, similar to the other tables